ofl/jainipurva/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,27 @@
+<p>
+    Jaini and Jaini Purva (both Devanagari) are typefaces based on the calligraphic style of the Jain Kalpasutra
+    manuscripts,
+    particularly referencing a manuscript from 1503 CE. The manuscript style has several unique features not seen in the
```

Comment:
@yanone, the formatting of the description file looks broken, please adjust it following this https://github.com/fonttools/fontbakery/issues/2811#issuecomment-1777309248

Reply:
This is really annoying given that the HTML rendering is unaffected by this. My text editor has formatted in this way, and I would have to manually reformat each HTML file after saving. I request that a VSCode plugin be provided by Google Fonts instead that formats HTML files in exactly the way intended by the issue.

Reply:
How is this broken? There is so much dogma floating around atm.

Reply:
There is not a check yet (the open issue requiring one has been linked above), but Dave has required us to fix this many times, and sometimes he has fixed it himself on Description files after merging. 

cc @davelab6

Reply:
We should only enforce checks which exist. Otherwise we can just pick any issue in https://github.com/fonttools/fontbakery/issues.

Reply:
> Dave has required us to fix this many times, and sometimes he has fixed it himself on Description files after merging

Please note that this has already been enforced in practice.

Reply:
If this is such a huge issue, we could also contemplate to add a one-liner to the packager that reformats the HTML file in an authoritative and reproducible way for every single PR, so that whatever user formatting is coming in will all be shaped into a single way.

Reply:
I'll do it in a min https://github.com/googlefonts/gftools/issues/817